statistics.md

```diff
@@ -26,7 +26,30 @@ The sections below describe the parameters for a page view event. All other info
   "targetType": "item"
 }
 ```
+## Report View Events endpoint
+**GET /api/statistics/viewevents**
```

Comment:
I don't think this endpoint is the right place for statistics reports. Since you use it to POST one event at a time, you would expect to GET one event at a time as well, and that's not very useful for a report.

I envision reports will get a dedicated endpoint e.g. `/api/statistics/reports` that will return data in some json equivalent of a table format

Reply:
I agree. That was my mistake to leave the endpoint as is. I will change it.